skyportal/models/schema.py

```diff
@@ -157,7 +157,7 @@ def add_schema(schema_class_name, exclude=[], add_to_model=False):
 
             schema_class_name = class_.__name__
             add_schema(
-                schema_class_name, exclude=['created_at', 'modified'], add_to_model=True
+                schema_class_name, exclude=['created_at', 'modified', 'healpix'], add_to_model=True
```

Comment:
This should work -- can you ensure that the ranges are not present in any other columns of any tables by a name other than `healpix`?

Reply:
Note that this also needs to be added to all `add_schema` calls below (that's not the problem, however, as I did that locally and am still seeing the ModelConversionError).

Reply:
What are all the columns that use the new ranges data type?

Reply:
@acrellin It's just the Tile: https://github.com/lpsinger/healpix-alchemy/blob/multiranges/healpix_alchemy/types.py#L32
So https://github.com/mcoughlin/skyportal/blob/fields2/skyportal/models/instrument.py#L194 I think?